--- nginx-1.10.0/debian/rules	2016-04-26 14:51:14.000000000 +0000
+++ /tmp/rules	2016-07-18 13:50:47.893850799 +0000
@@ -102,6 +102,10 @@
 			--add-module=$(MODULESDIR)/nginx-dav-ext-module \
 			--add-module=$(MODULESDIR)/nginx-echo \
 			--add-module=$(MODULESDIR)/nginx-upstream-fair \
+			--add-module=/opt/ngx_aws_auth \
+			--add-module=/opt/ngx_devel_kit \
+                       --add-module=/opt/lua-nginx-module \
+                       --add-module=/opt/set-misc-nginx-module \
 			--add-module=$(MODULESDIR)/ngx_http_substitutions_filter_module
 
 extras_configure_flags := \
